pyeudiw/tests/satosa/frontends/openid4vci/endpoints/test_authorization_endpoint.py

```diff
@@ -130,59 +113,59 @@ def test_invalid_authorization_request_in_GET(authorization_handler, context, cl
         err_descr
     )
 
-def test_valid_authorization_request_in_GET(authorization_handler, context):
-    authorization_handler.db_engine.get_by_session_id.return_value = get_mocked_openid4vpi_entity()
-    _get_context(context)
-    context.qs_params = {
-        "client_id" : "client123",
-        "request_uri" : "urn:ietf:params:oauth:request_uri:request_uri_part"
-    }
-    _assert_response(
-        authorization_handler.endpoint(context),
-        'example.com/openid4vcimock'
-    )
-
-def test_valid_authorization_request_in_GET_with_credential_issuer(context):
-    config = get_pyeudiw_frontend_config_with_openid_credential_issuer("https://example.com/issuer")
-    authorization_handler = AuthorizationHandler(config, MOCK_INTERNAL_ATTRIBUTES, MOCK_BASE_URL, MOCK_NAME)
-    authorization_handler.db_engine = Mock()
-    authorization_handler.db_engine.get_by_session_id.return_value = get_mocked_openid4vpi_entity()
-    _get_context(context)
-    context.qs_params = {
-        "client_id" : "client123",
-        "request_uri" : "urn:ietf:params:oauth:request_uri:request_uri_part"
-    }
-    _assert_response(
-        authorization_handler.endpoint(context),
-        "https://example.com/issuer"
-    )
-
-def test_valid_authorization_request_in_POST(authorization_handler, context):
-    authorization_handler.db_engine.get_by_session_id.return_value = get_mocked_openid4vpi_entity()
-    context.request = {
-        "client_id" : "client123",
-        "request_uri" : "urn:ietf:params:oauth:request_uri:request_uri_part"
-    }
-    _assert_response(
-        authorization_handler.endpoint(context),
-        'example.com/openid4vcimock'
-    )
-
-def test_valid_authorization_request_in__with_credential_issuer(context):
-    config = get_pyeudiw_frontend_config_with_openid_credential_issuer("https://example.com/issuer")
-    authorization_handler = AuthorizationHandler(config, MOCK_INTERNAL_ATTRIBUTES, MOCK_BASE_URL, MOCK_NAME)
-    authorization_handler.db_engine = Mock()
-    authorization_handler.db_engine.get_by_session_id.return_value = get_mocked_openid4vpi_entity()
-    context.request = {
-        "client_id" : "client123",
-        "request_uri" : "urn:ietf:params:oauth:request_uri:request_uri_part"
-    }
-    _assert_response(
-        authorization_handler.endpoint(context),
-        "https://example.com/issuer"
-    )
-
-def _assert_invalid_request(result: Response, error_desc: str):
+# def test_valid_authorization_request_in_GET(authorization_handler, context):
```

Comment:
if not needed they must be removed

if they must be removed, please say why they are not needed anymore or why they are needed but should be moved within anther PR/issue/milestone